packages/litexa/src/parser/state.coffee

```diff
@@ -307,7 +307,7 @@ class lib.State
       continue if name == '--default--'
       continue if intent.referenceIntent?
       unless intent.hasUtterances
-        unless name in extendedEventNames
+        unless context.skill.testDevice? and (name in extendedEventNames or name.includes('.')) # supported events have '.' in their names
           console.warn "`#{name}` does not have utterances; not adding to language model."
```

Comment:
Is there anything better than checking for existence of `testDevice` (thought there was something like `localTesting`)?

Regarding the other 2 checks: Do all event names have periods? Vice versa, couldn't a user declare a custom intent name with a period in it?

Reply:
I poked around what I could gather from context (a variable passed into here) and this seemed like the best one to use.

Users are not allowed to declare intents with periods in them unless they're one of the builtin intents. Dev console/SMAPI errors on this. I thought Litexa errors on this and you had to declare them explicitly as a supported intent, but now I'm not seeing that behavior...

Reply:
Maybe that was only for directives - the directiveWhitelist in litexa config.

Reply:
Found it in intent.coffee - will fix and add a test.

Reply:
#109 for the issue.